assets/src/dashboard/components/dialog/stories/index.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { action } from '@storybook/addon-actions';
+import { boolean, text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  Text,
+  THEME_CONSTANTS,
+} from '../../../../design-system';
+import Dialog from '../dialog';
+
+export default {
+  title: 'Dashboard/Components/Dialog/Base',
+  component: Dialog,
+};
+
+export const _default = () => {
+  return (
+    <Dialog
```

Comment:
Question: Are these still needed? Should we start getting rid of the old storybook things - `Dashboard`, `Playground`, and `Stories Editor`?

Reply:
great question. figured with the structure of `actions` within the dashboard and editor's dialogs that they were a good check on the UI.